gulp/plugins/util/parseType.js

```diff
@@ -0,0 +1,35 @@
+require('babel-register')
+const _ = require('lodash')
+const SUI = require('../../../src/lib/SUI') // eslint-disable-line no-unused-vars
+
+const evalValue = (values) => _.uniq(eval(values)).map(value => ({ // eslint-disable-line no-eval
+  computed: true,
+  value: `'${value}'`,
+}))
+
+const transformValues = (items) => _.flatMap(items, item => {
+  const { value } = item
+
+  if (_.startsWith(value, '...SUI')) return evalValue(value.substring(3))
+  return item
+})
+
+const parseEnum = type => {
```

Comment:
It looks like this reduces the need for some of the logic in `ComponentProps.js`:

[expandEnums()](https://github.com/Semantic-Org/Semantic-UI-React/blob/master/docs/app/Components/ComponentDoc/ComponentProps.js#L116)
[renderEnums()](https://github.com/Semantic-Org/Semantic-UI-React/blob/master/docs/app/Components/ComponentDoc/ComponentProps.js#L124)